web/src/main/java/org/mskcc/cbio/portal/web/ProxyController.java

```diff
@@ -54,7 +54,7 @@
   public void setBitlyURL(String property) { this.bitlyURL = property; }
 
   private String sessionServiceURL;
-  @Value("${session.service.url}")
+  @Value("${session.service.url:}")
```

Comment:
@inodb I think GlobalProperties is created for JSP pages mainly, if we are not going to use JSP anymore, we can easily use @Value("${property.name}") in whatever class we want to use that property, so no need to have a central class for managing properties.